package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "vue-form-generator",
 	"version": "3.0.0-dev",
```

Comment:
should be 2.2.3 or 2.30 - I'm thinking 2.3.0 due to all the changes?

Reply:
Yes, sorry. It was the v3 branch initially but I actually only made backward compatible changes. So I will call it 2.3.0, it will require `npm i` for all the collaborator since the test and plugin are updated, but it will change nothing for the end user. I tested it on my project without any problem and I use fairly complex custom fields, an api to generate the schema, complex dynamic functionality and they needed no changes and introduce no bug.
I might have missed something of course, but I'm pretty confident.